### ISSUE TITLE
Feature request: Library version with *no* SIMD impls

### ISSUE THREAD

**Issue by mumbleskates** [XKCP/K12#8]
I've been playing with ARM, which neither has SSSE3/AVX*, nor (understandably) does it seem to usually ship with compilers that understand the intrinsics or assembly.

Regardless, the K12 opt64 implementation seems to run about 33% faster than (for example) digestpp and it seems it would be handy to have a build mode for K12 that totally eschews intrinsics and assembly for running on more diverse systems.

Is this something you've already tried, and (for example) Inplace32BI happens to be just as fast or something? If it sounds like a good idea I would also be happy to pick this up.

**Comment by gvanas** [comment]
Yes, it does sound like a good idea to have a plain 64-bit target. It should be faster than `inplace32bi` on 64-bit systems.

If you want to do this, that's great, otherwise let me know and will do it.

What I would propose is to add a new fragment to `Makefile.build` like the following one, and exclude everything specific to x86_64 using the new define:
```xml
    <fragment name="plain64" inherits="optimized">
        <c>lib/Optimized64/KeccakP-1600-opt64.c</c>
        <h>lib/Optimized64/KeccakP-1600-SnP.h</h>
	<define>KeccakP1600_use_only_opt64</define>
    </fragment>
```

**Comment by mumbleskates** [comment]
From what I've been looking at it, it looks like it might be cleaner to have a distinct SnP header without all the extra symbols and higher alignment requirements, and possibly make a boundary cut between the opt64 implementation and the calls that dispatch to them, since they are currently lumped into the same file. (The only concern is this splits the dispatch across different TUs; might that matter? it doesn't seem like it but it's also plausible to look into a unity build.)

I'll see if I can make a stab at it later.

**Comment by gvanas** [comment]
OK, it makes sense, I agree with your suggestion.

**Comment by gvanas** [comment]
Implemented in #9
